frontend/src/metabase/actions/containers/ActionCreator/ActionCreatorHeader.styled.tsx

```diff
@@ -0,0 +1,59 @@
+import styled from "@emotion/styled";
+import { color } from "metabase/lib/colors";
+import { space } from "metabase/styled-components/theme";
+
+import Select from "metabase/core/components/Select";
+import SelectButton from "metabase/core/components/SelectButton";
+import EditableTextBase from "metabase/core/components/EditableText";
+
+export const Container = styled.div`
+  display: flex;
+  flex: 0 0 auto;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  background-color: ${color("white")};
+  border-bottom: 1px solid ${color("border")};
+  padding: ${space(2)} ${space(3)};
+`;
+
+export const LeftHeader = styled.div`
+  display: flex;
+  align-items: center;
+  color: ${color("text-medium")};
+  & > * ~ * {
```

Comment:
nit: any chance we can get rid of `*` selector? Maybe we can use flex with `gap` somewhere?

Reply:
💯  f96301243442999b395bb0763be0c581003e32a6